modules/table/src/test/java/org/apache/ignite/internal/table/RecordBinaryViewOperationsTest.java

```diff
@@ -342,6 +342,24 @@ public void getAll() {
         assertThat(res, contains(rec1, null, rec3));
     }
 
+    @Test
+    public void testContains() {
+        SchemaDescriptor schema = schemaDescriptor();
+        RecordView<Tuple> tbl = createTable(schema).recordView();
+
+        final long keyId = 1L;
+        Tuple rec = Tuple.create()
+                .set("id", keyId)
+                .set("val", 11L);
+        Tuple keyRec = Tuple.create()
+                .set("id", keyId);
+
+        tbl.insert(null, rec);
+
+        assertTrue(tbl.contains(null, keyRec));
+        assertFalse(tbl.contains(null, Tuple.create().set("id", -1L)));
```

Comment:
```suggestion
        assertTrue(tbl.contains(null, key));
        assertTrue(tbl.contains(null, keyRec));
        assertFalse(tbl.contains(null, Tuple.create().set("id", -1L)));
        assertFalse(tbl.contains(null, Tuple.create().set("id", -1L).set("val", -1L)));
```

Reply:
Overall looks good. 
Let's add a small fix to make test coverage similar to non-binary version.

Reply:
Hm, I tried it without a thought, but later I got that we can't do it: binary record view demands key only tuple as argument. Internally marshaller expected the same kind of input, from `TupleMarshaller.java`:

```java
    /**
     * Marshal tuple key part only.
     *
     * @param tuple Record tuple with key columns only.
     * @return Table row with columns set from given tuples.
     * @throws TupleMarshallerException If failed to marshal tuple.
     */
    Row marshalKey(Tuple tuple) throws TupleMarshallerException;
```

I may guess that there is the strange API situation (in pojo case we can marshall whole pojo with key-value splitting, but for a reasons with tuple marshaller we must pass key-only tuple).

Reply:
And I checked: there now no differences with [9627772](https://github.com/apache/ignite-3/pull/3118/commits/9627772996f4cf481f2a9447c8a7aba65dc58378), but check failed. I can't see logs of TC, it shows for me 404 :<

Reply:
@AMashenkov should I make any other updates there?